src/runtime/recipe/recipe.ts

```diff
@@ -701,10 +701,9 @@ export class Recipe implements Cloneable<Recipe> {
     return slot;
   }
 
-  get isLongRunning(): boolean {
-    return this.triggers.some(group =>
-        group.some(trigger => trigger[0] === 'launch' && trigger[1] === 'startup')
-        && group.some(trigger => trigger[0] === 'arcId' && !!trigger[1]));
+  getTrigger(group: [string, string][], name: string): string | null {
```

Comment:
We're inside of the recipe; can we avoid having to pass in `group`?

Reply:
i'm not really happy with this either, but...
i decided to move all the long-running arcs specific logic outside of recipe (because this is not a core Arcs concept).
in the helper methods inside storage-key-recipe-resolver i'm doing various checks for triggers that involve finding combination of key-value within a certain group. so i didn't find a better way to do it.
in general, i think we should create a class for `triggers` with appropriate helper methods, but i'm deferring this until we know more about how we want to use them.

Reply:
I like the eventual plan of creating a `triggers` class. 
If these tests aren't part of the core arcs concepts, then maybe we should remove `getTrigger` and let people iterate through `recipe.triggers` themselves? I think successive calls to `find` are a bit wasteful. 
Feel free to skip this suggestion, since it's such a small optimization.

Reply:
moved getTrigger as local helper method var in storage-key-rr (btw i don't like the name of that file). i'm not concerned about the `find`s, there only are handful of triggers (in fact ATM the long running arc indication is the only triggers we support!), so is not an issue. but the code is cleaner.

Reply:
I am not attached to the name of the file. If you have a better name, feel free to use that!